.github/workflows/release.yml

```diff
@@ -36,11 +36,22 @@ jobs:
         with:
           # This should be the same as the one specified for on.pull_request.branches
           ref: master
-      - name: Prepare cargo environment
-        shell: bash
-        run: ./.github/workflows/release/pre-publish.sh
-      - name: Publish to Crates
-        env:
-          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-        shell: bash
-        run: ./.github/workflows/release/publish.sh
+          token: ${{ secrets.GH_TOKEN }}
+      - name: Checkout actions
+        uses: actions/checkout@v3
+        with:
+          repository: pubnub/client-engineering-deployment-tools
+          ref: v1
+          token: ${{ secrets.GH_TOKEN }}
+          path: .github/.release/actions
+      - name: Publish to CocoaPods Trunk
```

Comment:
What is the purpose of this action?

Reply:
Is it ok now?

Reply:
I didn't requested for change. I was curious what exactly was this action doing. ;d
Anyway made change is also good. ;d

Reply:
It is an action, which will push to crates and run all our logic with notifications and other stuff :)